src/cli.rs

```diff
@@ -56,6 +58,11 @@ impl Options {
                 "-t" | "--title" => {
                     args_iter.next().map(|t| options.title = t);
                 },
+                "-e" => {
+                    // TODO: how do we handle `alacritty -e bash --login`, should all args
+                    //       given after -e be passed to the shell?
```

Comment:
I think we should try to emulate what other terminals do in this case. For example [urxvt](https://linux.die.net/man/1/urxvt) requires `-e` to be last, and all subsequent arguments are passed to the shell.